app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -1,5 +1,260 @@
-import React from 'react';
+import {
+  Area,
+  Button,
+  ButtonVariant,
+  Col,
+  FormikCheckbox,
+  FormikDropdown,
+  FormikText,
+  IOptionItem,
+  OptionItem,
+  Row,
+  Tab,
+  TabContainer,
+} from 'components';
+import { Formik } from 'formik';
+import { IContentApi, useApiEditor } from 'hooks';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLookup } from 'store';
 
+import { PropertiesContentForm } from '.';
+import { formContentToApiContent } from './contentFormUtils';
+import { TranscriptContentForm } from './TranscriptContentForm';
+
+/**
+ * Content Form edit and create form for default view. Path will be appeneded with content id.
+ * @returns Edit/Create Form for Content
+ */
 export const ContentForm: React.FC = () => {
-  return <div>Snippet Form</div>;
+  const [active, setActive] = useState('properties');
+  const navigate = useNavigate();
+  const {
+    storeMediaTypes,
+    state: { mediaTypes },
+  } = useLookup();
+  const [mediaTypeOptions, setMediatTypeOptions] = useState<IOptionItem[]>([]);
+  const api = useApiEditor({ baseURL: `/api/editor/contents/` });
+  const lookupApi = useApiEditor({ baseURL: '/api' });
```

Comment:
You don't need to specify the `baseURL` here.  It should be set by default.  The property is only there for override.